sa/sa.go

```diff
@@ -28,8 +28,8 @@ import (
 	sapb "github.com/letsencrypt/boulder/sa/proto"
 )
 
-type certCountFunc func(domain string, earliest, latest time.Time) (int, error)
-type getChallengesFunc func(authID string) ([]core.Challenge, error)
+type certCountFunc func(sel dbSelector, domain string, earliest, latest time.Time) (int, error)
```

Comment:
nit: Can we decide on a convention for what to name the variables holding these things, sometimes it's `tx`, sometimes `db`, and sometimes `sel`.